src/components/PivotTable/PivotTableColumnHeaderCell.js

```diff
@@ -30,24 +30,41 @@ export const PivotTableColumnHeaderCell = ({
                     header.span === 1 &&
                     engine.isSortable(index)
 
+                const style = {
+                    cursor: isSortable ? 'pointer' : 'default',
+                    width,
+                    maxWidth: width,
+                    minWidth: width,
+                }
+
+                if (
+                    engine.options.fixedColumnHeaders ||
+                    engine.options.fixedRowHeaders
+                ) {
```

Comment:
Shouldn't this condition just be `if (engine.options.fixedColumnHeaders)` ?

Reply:
🤔 I tried removing the or condition, and it seems to work the same, so yes, I don't think it's needed.
Removed.

Reply:
I see now why it didn't make a difference, it does set `top` and `left` on the column header cells, but the `position` is not set to fixed when the `fixedColumnHeaders` option is not passed.